rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -251,6 +253,14 @@ private class InMemHotStore[F[_]: Sync, C, P, A, K](
         )
       )
 
+  def toMap: F[Map[Seq[C], Row[P, A, K]]] =
+    for {
+      cache         <- S.read
+      data          = cache.data.readOnlySnapshot().map { case (k, v) => (Seq(k), v) }.toMap
```

Comment:
Tuple2 is a Bifunctor so this should work:
```data          = cache.data.readOnlySnapshot().leftMap(Seq(_)).toMap```

Reply:
Nope, looks like `leftMap` is part of either syntax.

```[error] [E1] rspace/src/main/scala/coop/rchain/rspace/HotStore.scala
[error]      value leftMap is not a member of scala.collection.Map[C,Seq[coop.rchain.rspace.internal.Datum[A]]]
[error]      L259:       data          = cache.data.readOnlySnapshot().leftMap(Seq(_)).toMap
[error]                                                                ^
```

Reply:
I was sloppy when writing the code in my original comment. Sorry about that!

The leftMap was supposed to be run on the Tuple2, not on the result of the readOnlySnapshop. This should work:
```data          = cache.data.readOnlySnapshot().map(_.leftMap(Seq(_))).toMap```

Reply:
Exactly